examples/mem-alloc-test/src/main.rs

```diff
@@ -103,22 +101,22 @@ fn wakeup_all_child_threads(pair_clone: Arc<(Mutex<bool>, Condvar)>) {
     drop(started);
 }
 
-fn traverse_buffer(buf: &mut Vec<i32>, scan_interval: usize) {
-    let mut ptr = 0;
+fn traverse_buffer(buf: *mut u8, size: usize, _scan_interval: usize) {
```

Comment:
Is it intentional that scan_interval is no longer used? Should the parameter just be removed instead?